plugins/auth/index.js

```diff
@@ -83,13 +85,20 @@ exports.register = (server, options, next) => {
      * Generates a jwt that is signed and has a 2h lifespan
      * @method generateToken
      * @param  {Object} profile Object from generateProfile
+     * @param  {Integer} expires_sec JWT Expires time (must be seconds)
```

Comment:
`expires_sec` is missing, maybe `expiresSec`

Reply:
Please align thes lines.

Reply:
I fixed!